### PR TITLE
Add user journeys

### LINKED ISSUE #17: Add content to user.html
Add content to user.html for the Ansible user persona.

### PR DESCRIPTION
**Pull request by oraNod**
Fixes #17 by adding content for user persona journeys

### CODE REVIEW

**Comment by oraNod** [comment]
@samccann and @Andersson007 Really struggling with some of the links for user actions here, especially the Galaxy and AWX docs....

**Comment by samccann** [comment]
how about using the galaxyNG docs instead? Those are likely better maintained than the old stuff we have on docs.ansible.com/ansible.

I think AWX only has individual RST files in their repo right? no docsite?

**Comment by samccann** [comment]
I think we're trying to do too much on this page. We have immediate 'calls to action' on the main landing page. For this user resources page, I think it would be helpful to have ONE pointer to the user guide (ansible/ansible) and then highlight all the other user project like you have here - galaxy, lint, navigator, etc. So my nickel - the point of this page should be to show users where they can get things beyond ansible-core so to speak. 

In general, the only 'wall of text' page should be the ecosystem page since there's no way to trim that down really.

File: data/pages.yaml [hunk 1]
```diff
@@ -53,6 +49,56 @@ novice:
...
+    intro: Write playbooks and develop content that automates complex tasks.
+    start: Start writing Ansible playbooks
+    evaluate_plugins: Learn which plugins to use for your tasks
+  build_inventory:
+    intro: Build inventories that list and describe your infrastructure.
+    multiple_hosts: Build inventory files to manage multiple hosts
+    dynamic: Use dynamic inventories
+  organize:
+    intro: Organize your automation project as it grows.
+    roles: Use roles to structure the automation project
```
Comment (samccann) [inline data/pages.yaml:75]: more details on what I mean by only using this page to point to other projects. I'd remove automate, build_inventory, and organize. They are covered on the front page.

File: data/pages.yaml [hunk 2]
```diff
@@ -53,6 +49,56 @@ novice:
...
+  tooling:
+    intro: Validate and improve your project for consistent, trusted automation.
+    lint: Use Ansible Lint to validate playbooks
```
Comment (samccann) [inline data/pages.yaml:78]: Consider removing all the intros - I think they add to the wall of text. And each link has it's own description so I'm not sure we gain much with the intros.

File: data/pages.yaml [hunk 3]
```diff
@@ -53,6 +49,56 @@ novice:
...
+# Resources for user journeys
+user:
+  title: User resources
+  intro: Automate the management of remote systems and control their desired state.
+  milestone:
+    automate: Create automation
+    build_inventory: Build inventories
+    organize: Organize automation projects
+    tooling: Use Ansible tooling
+    reuse: Re-use automation content
```
Comment (samccann) [inline data/pages.yaml:61]: Find automation content?  to distinguish this from creating roles (reusable content)

File: data/pages.yaml [hunk 4]
```diff
@@ -53,6 +49,56 @@ novice:
...
+    submit_roles: Submit roles to an existing collection
+    create_collection: Create a new collection
+    upload_collection: Upload a collection to Ansible Galaxy
+  awx:
+    intro: Use AWX for more convenient automation.
+    on_demand_jobs: Execute automation jobs on demand
+    schedule_jobs: Schedule automation jobs
+    create_job_templates: Create AWX job templates
+  executor:
+    intro: Invoke automation jobs from other systems.
```
Comment (samccann) [inline data/pages.yaml:99]: Is this the real purpose of EEs? I'd definitely change jobs as that seems an AWX term.  Are EEs more like portable containers for select versions of core, collections, and ..other bits... which of course does not make for a snappy intro...

File: data/pages.yaml (hunk 1 above)
Comment (oraNod) [reply data/pages.yaml:75]: @samccann agreed. thanks for the review comment here.

File: data/pages.yaml (hunk 2 above)
Comment (oraNod) [reply data/pages.yaml:78]: I'll do that in a separate issue so we only have to do it once without rebasing.

File: data/pages.yaml (hunk 4 above)
Comment (oraNod) [reply data/pages.yaml:99]: I'm no expert here but runner and sdk provide a programmatic interface to ansible so you can invoke automation runs from different systems or, say, a python application. runner/sdk call into to ansible runtime execution.

EEs give you a way to isolate dependencies for control nodes. e.g. you want to update databases you could use an ee with postgres drivers. @Spredzy could probably help check my understanding and make some clarifications.

File: data/pages.yaml (hunk 1 above)
Comment (oraNod) [reply data/pages.yaml:75]: https://github.com/ansible/jinja-docsite/issues/44

**Comment by oraNod** [comment]
> I think we're trying to do too much on this page. We have immediate 'calls to action' on the main landing page. For this user resources page, I think it would be helpful to have ONE pointer to the user guide (ansible/ansible) and then highlight all the other user project like you have here - galaxy, lint, navigator, etc. So my nickel - the point of this page should be to show users where they can get things beyond ansible-core so to speak.
> 
> In general, the only 'wall of text' page should be the ecosystem page since there's no way to trim that down really.

This sounds to me like it pulls away from the focus on journeys

File: data/links.yaml [hunk 5]
```diff
@@ -12,34 +12,38 @@ novice:
...
+  build_inventory:
+    multiple_hosts: "https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html"
+    dynamic: "https://docs.ansible.com/ansible/latest/inventory_guide/intro_dynamic_inventory.html"
+  organize:
+    roles: "https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html"
+  find:
+    explore_galaxy: "https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#finding-collections-on-galaxy"
+    install_roles: "https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-roles-from-galaxy"
+    install_collections: "https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-collections"
+    customize_content: ""
```
Comment (Andersson007) [inline data/links.yaml:33]:
```suggestion
    customize_content: ""
```
is it missed intentionally?

**Review by Andersson007 — COMMENTED** [review]
let's merge it and see how it'll look

File: data/links.yaml (hunk 5 above)
Comment (oraNod) [reply data/links.yaml:33]: @Andersson007 Yes I couldn't find that and will add it to the issue tracking content gaps. https://github.com/ansible/jinja-docsite/issues/32 

Have you got a suggestion for the link?

Comment (oraNod) [reply data/links.yaml:30]: I'll just add it to that issue and we can go from there.

**Pull request merged**